parser/parser.go

```diff
@@ -81,10 +81,11 @@ func ParseIntEntitlement(entitlementFormat string) (domain []string, id string,
 		return nil, "", 0, fmt.Errorf("Parsing of int entitlement %s failed: identifier must be alphanumeric and can contain '-'", entitlementFormat)
 	}
 
-	value, err = strconv.Atoi(valueString)
+	valueInt, err := strconv.Atoi(valueString)
```

Comment:
nit: you can use [`strconv.ParseInt`](https://golang.org/pkg/strconv/#ParseInt) directly and then avoid the type coercion below. The type coercion can panic, but it's fine in this codepath because it follows the `strconv.Atoi` call

```
value, err := strconv.ParseInt(valueString, 10, 64)
if err != nil {
  ...
}
return domain, id, value, nil
```

Reply:
Was looking for this, thank you!